exp/templates/exp/_navigation.html

```diff
@@ -12,23 +12,23 @@
 
     <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
       <ul class="nav navbar-nav navbar-right">
-        {% if user.is_authenticated %}
-        {% if user.is_superuser %}
-        <li><a href="{% url 'exp:organization-list' %}">Manage Organization</a></li>
+        {% if request.user.is_authenticated %}
+        {% if request.user.is_superuser %}
+        <li><a href="{% url 'exp:researcher-list' %}">Manage Organization</a></li>
         <li role="separator" class="divider"></li>
-        {% endif %} {# /if user.is_superuser #}
-        <li><a href="{% url 'exp:studies-list' %}">Manage Studies</a></li>
+        <li><a href="{% url 'exp:study-list' %}">Manage Studies</a></li>
+        {% endif %} {# /if request.user.is_superuser #}
         <li role="separator" class="divider"></li>
         <li class="dropdown">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
-                {{ user.identicon_small_html }} {{ user.get_short_name }} <span class="caret"></span>
+                {{ request.user.identicon_small_html }} {{ request.user.get_short_name }} <span class="caret"></span>
             </a>
             <ul class="dropdown-menu">
                 <li><a href="#">OSF Profile</a></li>
                 <li><a href="{% url 'logout' %}">Logout</a></li>
```

Comment:
I know you didn't do this, but after pulling this branch and loading experimenter I get a NoReverseMatch trying to render any page with this partial template in it. Changing it to `'web:logout'` makes it work fine.